docs/interactions/Receiving_and_Responding.md

```diff
@@ -69,10 +69,11 @@ While the `data` field is guaranteed to be present for all [interaction types](#
 ###### Message Component Data Structure
 
 | Field          | Type                                                                                                              | Description                                                                                                     |
-| -------------- | ----------------------------------------------------------------------------------------------------------------- | --------------------------------------------------------------------------------------------------------------- |
+|----------------|-------------------------------------------------------------------------------------------------------------------|-----------------------------------------------------------------------------------------------------------------|
 | custom_id      | string                                                                                                            | the [`custom_id`](#DOCS_INTERACTIONS_MESSAGE_COMPONENTS/custom-id) of the component                             |
 | component_type | integer                                                                                                           | the [type](#DOCS_INTERACTIONS_MESSAGE_COMPONENTS/component-object-component-types) of the component             |
 | values?\*      | array of [select option values](#DOCS_INTERACTIONS_MESSAGE_COMPONENTS/select-menu-object-select-option-structure) | values the user selected in a [select menu](#DOCS_INTERACTIONS_MESSAGE_COMPONENTS/select-menu-object) component |
+| resolved?      | [resolved data](#DOCS_INTERACTIONS_RECEIVING_AND_RESPONDING/interaction-object-resolved-data-structure)           | converted users + roles + channels + attachments                                                                |
```

Comment:
```suggestion
| resolved?      | [resolved data](#DOCS_INTERACTIONS_RECEIVING_AND_RESPONDING/interaction-object-resolved-data-structure)           | additional details about selected options                                                                |
```

Reply:
not super picky here, but we should abstract away "users + roles + channels + attachments" and save that for the details in the resolved docs section.

Reply:
Sorry for lack of an answer for a long period of time.

I understand, however, is "details" going to be appropriate here? I would personally rephrase it to be something like:
```suggestion
| resolved?      | [resolved data](#DOCS_INTERACTIONS_RECEIVING_AND_RESPONDING/interaction-object-resolved-data-structure)           | resolved payload entities                                                                                        |
```